firefox_puppeteer/ui/toolbars.py

```diff
@@ -331,6 +331,13 @@ def results(self):
         return self.popup.find_element('anon attribute',
                                        {'anonid': 'richlistbox'})
 
+    @property
+    def selected_index(self):
+        """
+        :returns: The index of the selected autocomplete result.
+        """
```

Comment:
Please keep the style of the doc as for other properties and add a short description. The :returns: can simply say 'The index'.